src/client/src/actions/wizardInfoActions/updateDependencyInfo.ts

```diff
@@ -0,0 +1,20 @@
+import { WIZARD_INFO_TYPEKEYS } from "./typeKeys";
+
+export interface IUpdateDependencyInfo {
+  type: WIZARD_INFO_TYPEKEYS.UPDATE_DEPENDENCY_INFO;
+  payload: any;
```

Comment:
Do you think it would be a good idea if we could had some type safety for this payload?